cluster/gce/coreos/helper.sh

```diff
@@ -46,6 +46,11 @@ function create-master-instance() {
   local address_opt=""
   [[ -n ${1:-} ]] && address_opt="--address ${1}"
 
+  local preemptible_master=""
+  if [[ "${PREEMPTIBLE_MASTER}" == "true" ]]; then
```

Comment:
Hi Mike

I played with having that var set or unset (on Mac - /bin/bash ). It seems to be fine either way. 

That syntax seems to expand to the same thing.  But I am happy to change it either way...